### ISSUE TITLE
False positive `Lift return` inspection on empty match

### ISSUE THREAD

**Issue by ortem** [intellij-rust/intellij-rust#5677]
## Environment

* **IntelliJ Rust plugin version:** 0.2.124.3122-201-nightly
* **Rust toolchain version:** 1.43.1 (8d69840ab 2020-05-04) x86_64-apple-darwin
* **IDE name and version:** IntelliJ IDEA 2020.1.1 Ultimate Edition (IU-201.7223.91)
* **Operating system:** macOS 10.15.3

## Problem description
`Lift return out of 'match'` inspection suggests wrong quick-fix on an empty match.

It transforms the following code
```Rust
fn foo() {
    match/*caret*/ true { // `Lift return` is available
    }
    let x = 1;
}
```
into
```Rust
fn foo() {
    return match/*caret*/ true {} // `Lift return` is available again but does nothing
    let x = 1;
}
```
